### ISSUE TITLE
Clean up aiofiles warnings on 3.8

### ISSUE THREAD

**Issue by simonw** [simonw/datasette#754]
https://travis-ci.org/github/simonw/datasette/jobs/682624476

Lots of warnings like this:
```
/home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/aiofiles/threadpool/utils.py:33

/home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/aiofiles/threadpool/utils.py:33

  /home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/aiofiles/threadpool/utils.py:33: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead

    def method(self, *args, **kwargs):

/home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/aiofiles/threadpool/__init__.py:27

  /home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/aiofiles/threadpool/__init__.py:27: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead

    def _open(file, mode='r', buffering=-1, encoding=None, errors=None, newline=None,
```

**Comment by simonw** [comment]
I think those will be fixed by an upgrade to aiofiles 0.5.0:

https://github.com/Tinche/aiofiles/compare/v0.4.0...v0.5.0

**Comment by simonw** [comment]
Sure enough this run https://travis-ci.org/github/simonw/datasette/jobs/674415040 from that PR no longer shows the warnings.
